pkg/metrics/get_workflow_runs_from_github.go

```diff
@@ -119,9 +119,10 @@ func getRelevantFields(repo string, run *github.WorkflowRun) []string {
 }
 
 func getRecentWorkflowRuns(owner string, repo string) []*github.WorkflowRun {
-	window_start := time.Now().Add(time.Duration(-12) * time.Hour).Format(time.RFC3339)
+	// FIXME: make the window dynamic
+	window_start := time.Now().Add(time.Duration(-8) * time.Hour).Format(time.RFC3339)
```

Comment:
why 4 less hours?

Reply:
Simply reducing the queries by %25.

I'll work today on reducing the API calls and make this range dynamic